admin_guide/cluster_management.rst

```diff
@@ -22,15 +22,14 @@ Adding a worker
 
 Citus stores all the data for distributed tables on the worker nodes. Hence, if you want to scale out your cluster by adding more computing power, you can do so by adding a worker.
 
-To add a new node to the cluster, you first need to add the DNS name of that node to the pg_worker_list.conf file in your data directory on the master node.
-
-Next, you can call the pg_reload_conf UDF to cause the master to reload its configuration.
+To add a new node to the cluster, you first need to add the DNS name of that
```

Comment:
Slight rephrasing. To add a new node to the cluster, you first need to add the DNS name of that node and port on which it is running postgres to the pg_dist_node catalog table.